search/retrieval-augmented-generation/examples/question_answering.ipynb

```diff
@@ -137,7 +141,7 @@
       },
       "outputs": [],
       "source": [
-        "! pip install google-cloud-aiplatform google-cloud-discoveryengine langchain==0.0.236 pydantic==1.10.8 typing-inspect==0.8.0 typing_extensions==4.5.0 --upgrade --user"
+        "%pip install google-cloud-aiplatform google-cloud-discoveryengine langchain pydantic==1.10.8 typing-inspect==0.8.0 typing_extensions==4.5.0 --upgrade --user"
```

Comment:
I suggest pinning the version of Cloud client libs and langchain lib to the latest version at the time of publishing, unless these are tested at a regular frequency (i.e., daily or weekly automated tests).

Reply:
I can do that for the Client Libs, and I'll do it for langchain once the new version is released with the changes.

Reply:
Sounds good, feel free to do in a followup PR so it's not blocking the other good updates in this one!